code/game/mecha/mech_fabricator.dm

```diff
@@ -147,6 +147,8 @@
 					sub_category += "Medical"
 				if(module_types & BORG_MODULE_ENGINEERING)
 					sub_category += "Engineering"
+			else if(U in typesof(/obj/item/borg/upgrade/modkit)) //added else if statment for cyborg items in mining_designs.dm
```

Comment:
Use ispath, not typesof. Also, why not just give modkits `module_types = BORG_MODULE_MINER`?

Reply:
because it is undefined, and if you try to define it, it just doesn't work. "module_types = " works in robot_upgrades.dm and does not work in mining.dm, that line of code is there, because mining.dm did not want to accept the var

Reply:
`module_flags`

Reply:
Yes, same statement I was making, but with the correct flag. What I meant. If you tell me how I can get mining.dm to accept the var definition, that can be done, It is what I was doing originally, but couldn't get it to work.

Reply:
Changed, removed code, and added module_types = BORG_MODULE_MINER to line 280 in the proper DM